pkg/gadgettracermanager/gadget-tracer-manager-maps.h

```diff
@@ -0,0 +1,37 @@
+#ifndef GADGET_TRACER_MANAGER_MAPS_H
+#define GADGET_TRACER_MANAGER_MAPS_H
+
+struct ns_common {
+	unsigned int inum;
+} __attribute__((preserve_access_index));
+
+struct mnt_namespace {
+	struct ns_common ns;
+}__attribute__((preserve_access_index));
```

Comment:
I'm a little bit worried what it'll create conflicts with files that include other headers defining these structures. Perhaps we could avoid that definition here and letting each tools to define / import a file defining it?

Reply:
Done :+1: